include/evmc/helpers.h

```diff
@@ -0,0 +1,52 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+/**
+ * EVMC Helpers
+ *
+ * A collection of helper functions for invoking a VM instance methods.
+ * These are convenient e.g. for Go bindings, because this is the only way to invoke a function
+ * via a pointer to function.
+ *
+ * @defgroup helpers EVMC Helpers
+ * @{
+ */
+#pragma once
+
+#include <evmc/evmc.h>
+
+/**
+ * Destroys the VM instance.
+ *
+ * @see evmc_destroy_fn
+ */
+inline void evmc_destroy(struct evmc_instance* instance)
```

Comment:
These should be `static inline` even though we have `#pragma once`. Not even sure we should have pragma once since that is more of a C++ism.

Reply:
`#pragma once` is not C++. Is... pragma. All compilers support it.

Reply:
Still, if we target FFI in languages which may not support pragma, can we just have `static` here since this in a header?